backend/routes/time.js

```diff
@@ -12,49 +12,57 @@ const processInput = require('../helpers/processInput');
 const timeQueries = require('../queries/time');
 
 //Gets the number of hours banked by a volunteer
-router.get('/past/:v_id', async (req, res, next) => {
+router.get('/hours/:v_id', async (req, res, next) => {
     try {
         const vId = processInput(req.params.v_id, "idNum", "volunteer id");
-        let allBankedHours = await timeQueries.allBankedHours(vId);
+        let allBankedHours = await timeQueries.allVolunteeredTimeByVolunteerID(vId);
 
-        res.json({
-            payload: allBankedHours,
-            message: "Success",
-            err: false
-        });
+        res.status(200)
+            .json({
+                payload: allBankedHours,
+                message: "Success",
+                err: false
+            });
     } catch (err) {
         handleError(err, req, res, next);
     }
 });
 
-//Gets the number of hours planned by a volunteer
-router.get('/upcoming/:v_id', async (req, res, next) => {
-    try {
-        const vId = processInput(req.params.v_id, "idNum", "volunteer id");
-        let allPlannedHours = await timeQueries.allPlannedHours(vId);
-        res.json({
-            payload: allPlannedHours,
-            message: "Success",
-            err: false
-        });
-    } catch (err) {
-        handleError(err, req, res, next);
-    }
-});
 
 //Gets the number of hours by all volunteers
 router.get('/', async (req, res, next) => {
     try {
         let allHours = await timeQueries.allHours();
 
-        res.json({
-            payload: allHours,
-            message: "Success",
-            err: false
-        });
+        res.status(200)
+            .json({
+                payload: allHours,
+                message: "Success",
+                err: false
+            });
     } catch (err) {
         handleError(err, req, res, next);
     }
 });
 
+router.patch('/update', async (req, res, next) => {
```

Comment:
because of our latest tables refactoring, updating the volunteer have been moved to event-volunteers table and updating the hours is part of eventAttendees router